pandas/util/_print_versions.py

```diff
@@ -43,7 +43,7 @@ def get_sys_info() -> List[Tuple[str, Optional[Union[str, int]]]]:
                 ("python-bits", struct.calcsize("P") * 8),
                 ("OS", f"{sysname}"),
                 ("OS-release", f"{release}"),
-                # ("Version", "{version}".format(version=version)),
+                # ("Version", f"{version}"),
```

Comment:
can you add a comment above "# FIXME: dont leave commented-out"

Reply:
Sure